databases/week3/mealsharing_database.sql

```diff
@@ -0,0 +1,36 @@
+DROP  DATABASE mealSharing;
+CREATE  DATABASE mealSharing;
+USE  mealSharing;
+
+CREATE TABLE `Meal` (
+  `id` int  unsigned auto_increment  primary key ,
+  `title` varchar(255) NOT NULL ,
+  `description` text NOT NULL ,
+  `location` varchar(255) NOT NULL ,
+  `time` datetime NOT NULL ,
```

Comment:
Just to follow the exact column names in the diagram, please change this to `when` :)